KsApi/models/CreatePaymentSourceEnvelope.swift

```diff
@@ -4,19 +4,7 @@ public struct CreatePaymentSourceEnvelope: Decodable {
   public var createPaymentSource: CreatePaymentSource
 
   public struct CreatePaymentSource: Decodable {
-    public var errorMessage: String?
     public var isSuccessful: Bool
-    public var paymentSource: GraphUserCreditCard.CreditCard
-  }
-}
-
-extension CreatePaymentSourceEnvelope {
-  private enum CodingKeys: String, CodingKey {
-    case createPaymentSource
-  }
-
-  public init(from decoder: Decoder) throws {
-    let values = try decoder.container(keyedBy: CodingKeys.self)
-    self.createPaymentSource = try values.decode(CreatePaymentSource.self, forKey: .createPaymentSource)
+    public var paymentSource: UserCreditCards.CreditCard
```

Comment:
Why did you remove the `errorMessage` property?

Reply:
The `errorMessage` property on the GQL mutation is deprecated, so no need to reference it here or in the mutation anymore.